turbo.json

```diff
@@ -7,7 +7,9 @@
     "insta.yaml",
     ".github/workflows/build.yml",
     "scripts/*",
-    "patches/*"
+    "patches/*",
+    "tools/bump-pack/**",
+    "!tools/bump-pack/node_modules/**"
```

Comment:
Should we ignore all `node_modules`?

Reply:
Shouldn't be needed since other globs are more specific, while `tools/bump-pack/**` isn't. Will double check others though.